platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -374,101 +369,91 @@ sep24:
   # `PUT /customer` endpoint of the business server to create or update the KYC fields.
   kyc_fields_forwarding:
+    # Whether to enable the KYC fields forwarding.
     enabled: true
 
+######################
+# SEP-31 Configuration
+######################
 sep31:
-  ## @param: enabled
-  ## @type: bool
-  ## Whether to enable SEP-31 for this instance of AP server.
+  ## Whether to enable SEP-31
   #
   enabled: false
-  ## @param: payment_type
-  ## @default: STRICT_SEND
-  ## Used to determine how amount_in is calculated from amount in the POST /transaction call
-  ## @supported_values:
-  ##     STRICT_SEND: amount_in = amount
-  ##     STRICT_RECEIVE: amount_in = amount + fee
+  # The `payment_type` must be one of the following:
+  #     STRICT_SEND: amount_in = amount
+  #     STRICT_RECEIVE: amount_in = amount + fee
   payment_type: STRICT_SEND
 
-  ## @param: deposit_info_generator_type
-  ## @default: self
-  ## Used to choose how the SEP-31 deposit information will be generated, which includes the
-  ##     deposit address, memo and memo type.
-  ## @supported_values:
+  # Used to choose how the SEP-31 deposit information will be generated, which includes the
+  #     deposit address, memo and memo type.
+  # Tje `deposit_info_generator_type` must be one of the following:
   ##     self: the memo and memo type are generated in the local code, and the distribution account is used for the deposit address.
   ##     circle: the memo and memo type are generated through Circle API, as well as the deposit address.
   ##     api: the memo and memo type are generated through calling the anchor's GET /unique_address endpoint.
   #
   deposit_info_generator_type: self
 
+######################
+# SEP-38 Configuration
+######################
 sep38:
-  ## @param: enabled
-  ## @type: bool
-  ## Whether to enable SEP-38 for this instance of AP server.
+  # Whether to enable SEP-38
   #
   enabled: false
 
 
 ##########################
-## Metrics Configuration
+# Metrics Configuration
 ##########################
-## The metrics are exposed at the port specified by sep_server.management_server_port
+# The metrics are exposed at the port specified by each server's (eg: sep, platform, event processor and observer)
+# management_server_port
 metrics:
-  ## @param: enabled
-  ## @type: bool
-  ## If true, enable metrics endpoints for third-party services (ex: prometheus) to scrape
-  #
+  # If true, enable metrics will be enabled.
   enabled: false
-  ## @param: extras_enabled
-  ## @type: bool
-  ## If true, enable extra metrics (these metrics periodically query the database to get the current state of the
-  ## system)
-  #
-  extras_enabled: false
-  ## @param: run_interval
-  ## @type: integer
-  ## Interval in seconds to query the database to generate metrics
-  #
-  run_interval: 30
+  # The configuration of making extended metrics available.
+  extended_metrics:
```

Comment:
Question: What are extended metrics?

Reply:
They are defined in https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/java/org/stellar/anchor/platform/service/MetricEmitterService.java#L100-L107

Reply:
These fields are subject to review and refactor.